.eslintrc

```diff
@@ -75,7 +80,33 @@
         "aspects": ["invalidHref", "preferButton"]
       }
     ],
-    "no-await-in-loop": "off"
+    "no-await-in-loop": "off",
+    "simple-import-sort/imports": [
+      "error",
+      {
+        "groups": [
+          // Side effect imports.
+          ["^\\u0000"],
+          // Packages.
+          // Things that start with a letter (or digit or underscore), or `@` followed by a letter.
+          ["^@?\\w"],
+          // Absolute imports and other imports such as Vue-style `@/foo`.
+          // Anything that does not start with a dot.
+          ["^[^. ]"],
+          // ALiases.
+          [
+            "^(@components|@paths|@pages|@utils|@hooks|@types|@app|@locale|@temp)(/.*|$)"
+          ],
+          // Relative imports.
+          // Anything that starts with a dot.
+          ["^\\."],
+          // Assets.
+          ["^(images|!!raw-loader)"],
+          // Styles
+          ["w*.*css|^@styles"]
```

Comment:
The `@styles` helper is mostly for style utilities, I suggest bundling it above with other absolute imports (aliases)